WireUI/Sources/WireDesign/Icons/Image+ImageResource.swift

```diff
@@ -21,4 +21,5 @@ import SwiftUI
 public extension Image {
     static let close = Image(.close)
     static let info = Image(.info)
+    static let infoCircle = Image(.infoCircle)
```

Comment:
I strongly encourage to not have feature specific image assets at a central point.
Please move it to the target where it is used.
If this should be a view component to be reused, please move it to `WireReusableUIComponents`.

Reply:
done